### ISSUE TITLE
OED Toolkits - MS Silverlight Depreciation on the Toolkits

### ISSUE THREAD

**Issue by HollySchuelerKCI** [kcigeospatial/sha_oed_issues#383]
**Original discussion occurred on 2/2 meeting with Tad, Viji, Scott and Holly to discuss Task 18 and Task 20.**

Original Request:

Tad expressed a new question regarding the deprecation of MS Silverlight on the Toolkits.

a.  Windows 10 & Microsoft edge and Silverlight will be deprecated

b. Currently Silverlight is utilized to support signatures exclusively on a desktop platform; other form factors such as phone and tablet platforms allow the signature

c. What is general technical process to port signature functionality off of Silverlight for desktop

**_Action Item: 	Scott/Holly will record the concerns regarding Silverlight within GH and pass onto solutions architect for an initial assessment_**

**Comment by Tad-Daniel** [comment]
I have noticed that the jQuery routine works properly on my tablet with MS Edge.  Could we cut off Silverlight in the developer environment and let me try that on several browsers.  If JQuery works well on all browsers we could push to production.  I currently have two win 10 units in the field and several more coming.  Let me know if this is possible.

**Comment by vijme** [comment]
Yes, HTML5/JQuery will be the way to address this issue. That was exactly what I was doing it yesterday morning, but had certain reference issues to be fixed. Now the dev is running without silverlight. Please note that there was some minor code changes and has been changed only for QA-1 for now. The button image is also not displayed correctly.  If you could test and confirm, we can go ahead and fix that and implement the changes in QA-2 and any other places where signature is captured. (I am not sure if there is any in other toolkits). Thanks.

**Comment by Tad-Daniel** [comment]
I tested it on multiple device and they all work well.  Just need to fix the button and QA-2
Safari iOS, win 10 edge, win 10 ie, win 10 chrome, safari MacOS, firefox iOS.

Those are the only two places that capture signatures

**Comment by vijme** [comment]
Looks like there is an existing bug in QA-2 that signatures are not saved when signed from mobile device or edge. Will be fixed.

**Comment by vijme** [comment]
Item working in development

**Comment by slmessier** [comment]
@Tad-Daniel & @HollySchuelerKCI 
Please test/re-test at earliest convenience and log the results of your testing. Thank you!

**Comment by slmessier** [comment]
@vijme - please list the specific instances of signature functionality that can be tested at the current time. Are there instances (such as QA2) that require an update after confirmation, or have all instances already been updated? Thank you!

**Comment by vijme** [comment]
Both QA-1 and QA-2 have been updated and ready for testing. (Item can be tested by viewing it against the STAGING environment where the signature screen for Silverlight is different in IE). Please note that in QA-2, "Signature not saved" text appears after save signature button, but the signature is saved indeed. I will remove the text shortly, once I get the access to the system.

**Comment by vijme** [comment]
The display text was removed. Ready for testing.

**Comment by HollySchuelerKCI** [comment]
This issue has been validated by the project team and confirmed by stakeholders. This issue is ready for staging.

**Comment by HollySchuelerKCI** [comment]
This issue has been validated by the project team in the Staging Environment.

**Comment by Tad-Daniel** [comment]
Working well

**Comment by HollySchuelerKCI** [comment]
This issue has been confirmed by the stakeholder and is ready for Production.

**Comment by Tad-Daniel** [comment]
Working well.  This item is complete
